action.yml

```diff
@@ -14,6 +18,9 @@ inputs:
   upload-chunk-size:
     description: 'The chunk size used to split up large files during upload, in bytes'
     required: false
+  new-primary-key-env-variable:
```

Comment:
change to `override-primary-key-env-variable`

Reply:
Done